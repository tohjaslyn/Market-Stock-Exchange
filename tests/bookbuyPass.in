add trader1 100
buy trader1 ABC 5 2
buy trader1 ABC 5 2.50
book buy
exit
