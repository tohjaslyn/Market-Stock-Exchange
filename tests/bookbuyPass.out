$ Success.
$ No trades could be made, order added to buy book.
$ No trades could be made, order added to buy book.
$ 0001: BUY 5.00xABC @ $2.00
0002: BUY 5.00xABC @ $2.50
$ Have a nice day.
